schema.sql

```diff
@@ -2,83 +2,105 @@
-CREATE TABLE owners (
+  CREATE TABLE owners (
     id int GENERATED ALWAYS AS IDENTITY,
     full_name varchar(200),
     age int
-);
+  );
 CREATE TABLE species (
     id int GENERATED ALWAYS AS IDENTITY,
     name varchar(100)
-);
+  );
 BEGIN
-  SELECT * FROM animals;
-  ALTER TABLE animals MODIFY id int GENERATED ALWAYS AS IDENTITY;
-  ALTER TABLE animals DROP COLUMN species;
-  ALTER TABLE animals ADD species_id int;
-  ALTER TABLE animals ADD CONSTRAINT FK_species_id FOREIGN KEY (species_id) REFERENCES species(id);
-  ALTER TABLE animals ADD owner_id int;
-  ALTER TABLE animals ADD CONSTRAINT FK_owners_id FOREIGN KEY (owner_id) REFERENCES owners(id);
-  SELECT * FROM animals;
+SELECT
+  *
+FROM
+  animals;
+ALTER TABLE
+  animals
+MODIFY
+  id int GENERATED ALWAYS AS IDENTITY;
+ALTER TABLE
+  animals DROP COLUMN species;
+ALTER TABLE
+  animals
+ADD
+  species_id int;
+ALTER TABLE
+  animals
+ADD
+  CONSTRAINT FK_species_id FOREIGN KEY (species_id) REFERENCES species(id);
+ALTER TABLE
+  animals
+ADD
+  owner_id int;
+ALTER TABLE
+  animals
+ADD
+  CONSTRAINT FK_owners_id FOREIGN KEY (owner_id) REFERENCES owners(id);
+SELECT
+  *
+FROM
+  animals;
 COMMIT;
 -- ======================week 1 day 4==========================
-CREATE TABLE vets (
+  CREATE TABLE vets (
     id int GENERATED ALWAYS AS IDENTITY,
     name varchar(200),
     age int,
     date_of_graduation date
-);
-ALTER TABLE vets ADD CONSTRAINT pk_vets PRIMARY KEY(id);
+  );
+ALTER TABLE
+  vets
+ADD
+  CONSTRAINT pk_vets PRIMARY KEY(id);
 -- Many-to-Many relationship table between species & vets
-CREATE TABLE specializations (
+  CREATE TABLE specializations (
     specie_id int NOT NULL,
     vet_id int NOT NULL,
-    CONSTRAINT fk_species
-      FOREIGN KEY(specie_id) 
-	  REFERENCES species(id)
-	  ON DELETE CASCADE,
-    CONSTRAINT fk_vets
-      FOREIGN KEY(vet_id) 
-	  REFERENCES vets(id)
-	  ON DELETE CASCADE
-);
+    CONSTRAINT fk_species FOREIGN KEY(specie_id) REFERENCES species(id) ON DELETE CASCADE,
+    CONSTRAINT fk_vets FOREIGN KEY(vet_id) REFERENCES vets(id) ON DELETE CASCADE
+  );
 -- Many-to-Many relationship table between animals & vets
-CREATE TABLE visits (
+  CREATE TABLE visits (
     animal_id int NOT NULL,
     vet_id int NOT NULL,
     date_of_visit date,
-    CONSTRAINT fk_animals
-      FOREIGN KEY(animal_id) 
-	  REFERENCES animals(id)
-	  ON DELETE CASCADE,
-    CONSTRAINT fk_vets
-      FOREIGN KEY(vet_id) 
-	  REFERENCES vets(id)
-	  ON DELETE CASCADE
-);
+    CONSTRAINT fk_animals FOREIGN KEY(animal_id) REFERENCES animals(id) ON DELETE CASCADE,
+    CONSTRAINT fk_vets FOREIGN KEY(vet_id) REFERENCES vets(id) ON DELETE CASCADE
+  );
+-- ======================week 2 day 1==========================
+ALTER TABLE
+  owners
+ADD
+  COLUMN email VARCHAR(120);
+CREATE INDEX idx_visits_animal_id ON visits(animal_id);
+CREATE INDEX idx_visits_vet_id ON visits(vet_id);
+CREATE INDEX idx_owners_email ON owners(email);
```

Comment:
- [ ] Kindly ensure your before query `SELECT * FROM owners where email = 'owner_18327@mail.com'` has more than  1000ms execution time, 
![image](https://user-images.githubusercontent.com/61408860/200912962-a17c795e-3eba-4f76-81db-b5579bae99e5.png)

- [ ] @ZikaZaki and @AlierPM  it would be nice if you tried and reduce the execution time in the query shown in the screenshot below
![image](https://user-images.githubusercontent.com/61408860/200913477-2ee5043f-060a-4918-a17f-3bbc6b3e1407.png)

Reply:
Hi @danmainah , It's already reduced.

Reply:
@ZikaZaki great